packages/features/ee/dsync/components/GroupTeamMappingTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { useRef, useState } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { DataTable, Button } from "@calcom/ui";
+
+import CreateTeamDialog from "./CreateTeamDialog";
+import GroupNameCell from "./GroupNameCell";
+
+const GroupTeamMappingTable = () => {
+  const { t } = useLocale();
+  const [createTeamDialogOpen, setCreateTeamDialogOpen] = useState(false);
+
+  const { data } = trpc.viewer.dsync.teamGroupMapping.get.useQuery({ orgId: 1 });
```

Comment:
```suggestion
  const { data } = trpc.viewer.dsync.teamGroupMapping.get.useQuery();
```
I think this can be removed, right?